eng/Version.Details.xml

```diff
@@ -22,11 +22,7 @@
       <Uri>https://github.com/dotnet/dotnet</Uri>
       <Sha>85778473549347b3e4bad3ea009e9438df7b11bb</Sha>
     </Dependency>
-    <Dependency Name="VS.Redist.Common.NetCore.SharedFramework.x64.10.0" Version="10.0.0-preview.5.25260.104">
-      <Uri>https://github.com/dotnet/dotnet</Uri>
-      <Sha>85778473549347b3e4bad3ea009e9438df7b11bb</Sha>
-    </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Runtime.win-x64" Version="10.0.0-preview.5.25260.104">
+    <Dependency Name="Microsoft.NETCore.Platforms" Version="10.0.0-preview.5.25260.104">
```

Comment:
Can we use Microsoft.NETCore.App.Ref instead? The platforms package currently ships as non-shipping but we might stop that at some point (nothing planned yet).

Reply:
Ah ok, makes sense. Yes I'm aware of these cases. Makes sense to me then. I would wish there would be another way to get that information without a sentinel dependency that could change.